src/js/components/PodSpecView.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import DescriptionList from './DescriptionList';
+import PodSpec from '../structs/PodSpec';
+import Icon from './Icon';
+
+const METHODS_TO_BIND = [
+];
+
+class PodSpecView extends React.Component {
```

Comment:
Yes, I am also in favour of the latter. There is a need for some general renaming of the old *Service stuff to *Application stuff.